File: swagger/models/upload_client_document_request.py
from .base_model import BaseModel

from .client_document import ClientDocument

class UploadClientDocumentRequest(BaseModel):
	"""
	no description available

	Attributes:

		client_id
		The RSSID of the client for whom the document is to be uploaded.

		file
		Contains information about the file to be uploaded.
	"""

	input_map = {
		'ClientId': 'client_id',
		'File': ('file', ClientDocument),
		}

	output_map = {
		'client_id': 'ClientId',
		'file': 'File',
		}

	client_id = None
	file = None



	def __init__(self, client_id, file):
		self.client_id = client_id
		self.file = file

File: swagger/Models/UploadClientDocumentRequest.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class UploadClientDocumentRequest extends BaseModel
{
	/**
	 * The RSSID of the client for whom the document is to be uploaded.
	 * 
	 * @var string
	 */
	public $clientId;
	/**
	 * Contains information about the file to be uploaded.
	 * 
	 * @var ClientDocument
	 */
	public $file;
	/**
	 * Constructor
	 */
	public function __construct($clientId, $file)
	{
		$this->clientId = $clientId;
		$this->file = $file;
	}

	protected function getInputMap()
	{
		return [
			'ClientId' => 'clientId',
			'File' => ['file', ClientDocument::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'clientId' => 'ClientId',
			'file' => 'File',
			];
	}
}
